font/src/ft/mod.rs

```diff
@@ -425,6 +448,31 @@ impl FreeTypeRasterizer {
         face.ft_face.load_glyph(index as u32, face.load_flags)?;
 
         let glyph = face.ft_face.glyph();
+
+        // Generate synthetic bold.
+        if face.embolden {
+            unsafe {
+                freetype_sys::FT_GlyphSlot_Embolden(glyph.raw()
+                    as *const freetype_sys::FT_GlyphSlotRec
+                    as *mut freetype_sys::FT_GlyphSlotRec);
+            }
+        }
+
+        // Transform glyphs with the matrix from Fontconfig. Primary used to generate italics.
+        if let Some(matrix) = face.matrix.as_ref() {
+            unsafe {
+                let glyph = face.ft_face.raw().glyph;
+
+                // Check that the glyph is outline.
```

Comment:
```suggestion
            let glyph = face.ft_face.raw().glyph;
            
            unsafe {
                // Check that the glyph is outline.
```

Moving this outside of `unsafe` should make it a little more clear what operations are actually problematic. Since I wasn't sure if `raw()` was safe or not.